tests/errors_test.py

```diff
@@ -42,3 +42,5 @@ def test_dj_exception() -> None:
 - Error 1 (error code: 0)
 - Error 2 (error code: 0)"""
     )
+
+    assert DJException("Message") == DJException("Message")
```

Comment:
This looks like `assert foo == foo` to me. What am I missing?

Reply:
This is to check that the `__eq__` method is working properly...and to get coverage on those lines :D